packages/apollo-voyager-offline/src/VersionedObjectState.ts

```diff
@@ -0,0 +1,28 @@
+import debug from 'debug'
```

Comment:
`import * as debug from 'debug'` to avoid the debug problem we have before?

Reply:
Debug will be gone soon from here.